distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -0,0 +1,243 @@
+from Standard.Base import all
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+
+from project.Errors import Invalid_Value_Type
+
+## Type to represent the different sizes of integer or float possible within a database.
+type Bits
+    ## 16-bit (2 byte) value
+    Bits_16
+    ## 32-bit (4 byte) value
+    Bits_32
+    ## 64-bit (8 byte) value
+    Bits_64
+
+    ## PRIVATE
+    to_bits : Integer
+    to_bits self = case self of
+        Bits.Bits_16 -> 16
+        Bits.Bits_32 -> 32
+        Bits.Bits_64 -> 64
+
+    ## PRIVATE
+    from_bits : Integer -> Bits
+    from_bits bits = case bits of
+        16 -> Bits.Bits_16
+        32 -> Bits.Bits_32
+        64 -> Bits.Bits_64
+        _ : Integer -> Error.throw (Illegal_Argument.Error "Invalid number of bits for a float or integer type.")
+
+    to_text : Text
+    to_text self = case self of
+        Bits.Bits_16 -> "16 bits"
+        Bits.Bits_32 -> "32 bits"
+        Bits.Bits_64 -> "64 bits"
+
+type Bits_Comparator
```

Comment:
This type is "private". Shouldn't it rather be defined in `Value_Type_Helpers`? It'd be more hidden there.